server/puppeteer-utils.js

```diff
@@ -51,7 +51,24 @@ const createImage = async (userId, challengeId, dir) => {
   }
 };
 
+const seedImage = async (fileName, dir) => {
```

Comment:
After your and my PRs are merged we should refactor the 3 `createImage` methods we have into one.

Reply:
@theo333 Agreed.